packages/db/src/migrations/0023_yeet_cognito.js

```diff
@@ -0,0 +1,19 @@
+import { sql } from "kysely"
+
+/** @param db {import('kysely').Kysely} */
+export async function up(db) {
+  await db.schema
+    .alterTable("ow_user")
+    .dropColumn("cognitoSub")
+    .addColumn("auth0Sub", sql`character varying(50)`, (col) => col.unique())
```

Comment:
```suggestion
    .addColumn("auth0Sub", sql`character varying(50)`, (col) => col.default(sql`null`).unique())
```

Reply:
Isn't that already the default? We should add a migration afterwards that add `.notNull()`.

Reply:
fair enough